dot/rpc/modules/gomock_reflect_2744729207/prog.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+
+package main
```

Comment:
This file looks auto-generated, looks like a result of running tests and closing them prematurely. I wonder if there are ways to create these files in temp directory instead of here.

Reply:
Yeah, they are generated by the go:generate command, I don't think it is possible to generate them in another folder, I will delete them